docs/datatypes/schedule.md

```diff
@@ -96,3 +96,9 @@ recur, using the minimum of the two delays between recurrences:
 ```scala mdoc:silent
 val expCapped = Schedule.exponential(100.milliseconds) || Schedule.spaced(1.second)
 ```
+
+Stops trying to retry after a specified amount of time has elapsed:
```

Comment:
Maybe just `Stops retrying after...`? WDYT?

Reply:
Yeah, I like that better. Updated.